packages/test/test-driver-definitions/src/interfaces.ts

```diff
@@ -46,35 +46,45 @@ export interface ITestDriver {
 	readonly version: string;
 
 	/**
-	 * Creates a document service factory targetting the server
+	 * Creates a document service factory targetting the server that corresponds to this driver.
 	 */
 	createDocumentServiceFactory(): IDocumentServiceFactory;
 
 	/**
-	 * Creates a url resolver targetting the server
+	 * Creates a url resolver targetting the server that corresponds to this driver.
 	 */
 	createUrlResolver(): IUrlResolver;
 
 	/**
-	 * Creates a create new request based on the test id.
+	 * Creates a 'create new' request appropriate for the server that corresponds to this driver.
+	 *
+	 * @remarks
 	 * Repeated calls with the same test id will return the same request.
 	 * The test id may not map directly to any specific Fluid Framework concept.
 	 * If you need more control you should disambiguate the driver based on its
-	 * type, this should only be done it absolutely necessary for complex scenarios
+	 * type; this should only be done it absolutely necessary for complex scenarios
 	 * as the test may not work against all supported servers if done.
+	 *
+	 * @param testId - Implementations can use this to try to generate a consistent request
```

Comment:
Nit: is "try" the right word here? I'm assuming it is guaranteed to be consistent if the same ID is passed?

Reply:
Probably worth noting the default behavior here too.

Reply:
Yeah good point. I wasn't super familiar with the implementations but I just looked through them and seems like yes, they all guarantee it and that's probably the whole intention. I'll update it.